source/src/Human.kt

```diff
@@ -0,0 +1,23 @@
+open class Human(var speed: Int =0, var weight: Int =0, var height: Int = 0, var gender: String = "", var Interface: IWalker? = null ){}
+
+object HumanFactory{
+    fun generateNewHuman (): Human{
+        val human: Human = Human()
+        val randomChoice = (1..2).random()
+        if (randomChoice == 1) {
+            human.gender = "Male"
+            human.weight = (60..200).random()
+            human.height = (160..220).random()
+            human.speed = (1..15).random()
+        }
+        else{
+            human.gender = "Female"
+            human.weight = (40..60).random()
+            human.height = (150..180).random()
+            human.speed = (1..10).random()
+        }
+        human.Interface = SheeshWalker
```

Comment:
Now that variable is pointer to object, but it's must be a var
Try this
```
human.walker = SheeshWalker()
```
I've already replaced name of variable(Interface -> walker)